index.js

```diff
@@ -85,10 +79,19 @@ const earningsCommand = require('./commands/earnings')
 const valueCommand = require('./commands/value')
 const walletCommand = require('./commands/wallet')
 
-statsCommand.init(controller)
-earningsCommand.init(controller)
-valueCommand.init(controller)
-walletCommand.init(controller)
+/*********
+ * Chatter
+ *********/
+const random = require('./commands/random')
```

Comment:
So order here matters. There's an addition in the next commit that causes a bug which I'll fix in a future commit (not today).

Basically, I added a catch all command. That command tends to catch before others (stupid). Anyhow, just wanted to let you know I'm aware of it and I'll fix it later. :)